txcache/transactionsTracker.go

```diff
@@ -0,0 +1,127 @@
+package txcache
+
+import (
+	"math"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+)
+
+// accountRange defines a structure which contains the minimum and the maximum nonce of an account
+type accountRange struct {
+	minNonce core.OptionalUint64
+	maxNonce core.OptionalUint64
+}
+
+// transactionsTracker contains a map of accounts and for each account corresponds a (minNonce, maxNonce) range
+type transactionsTracker struct {
+	accountsWithRange map[string]*accountRange
+}
+
+// newTransactionsTracker creates a new transactions tracker
```

Comment:
In the comments, explain what does `transactions` mean - what transactions should be passed.

Now I realize, maybe `selectionTracker` should be called `proposedBlocksTracker`?